src/ingest/als_nmr.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime
```

Comment:
I would like to move this file into the `nmr_schema` package, and may all it "ingest_nmr.py".

Reply:
I dont fully understand this comments. Do you mean we move all files in "ingest" folder to "nmr_schema" folder?